mfrc522/iso14443com.py

```diff
@@ -0,0 +1,55 @@
+import crcmod
+from .mfrc522 import *
+import struct
+
+SEL_CASCADE_1   = 0x93
+SEL_CASCADE_2   = 0x95
+SEL_CASCADE_3   = 0x97
+
+__crc_func = crcmod.mkCrcFun(0x11021, initCrc=0x6363)
+
+def __calculate_crc(data):
+    return struct.pack('<H', __crc_func(data))
+
+def __perform_cascade(module, cascade_level):
```

Comment:
Rename `module` to `rfid_module` or something, the word "module" can mean just about anything.